src/main/java/com/byznass/tiolktrack/postgres/dao/PostgresLocationPersister.java

```diff
@@ -25,29 +25,30 @@ public PostgresLocationPersister(Connection connection) {
 	}
 
 	@Override
-	public void persistLocation(Location location) {
+	public void persistLocation(Location location) throws TiolkTrackException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)